tests/santa/test_santa_api_views.py

```diff
@@ -724,13 +727,49 @@ def test_eventupload_bundle_binary(self, post_event):
     # postflight
 
     def test_rule_postflight_not_enrolled(self):
-        url = reverse("santa:postflight", args=(self.enrollment_secret.secret, uuid.uuid4()))
+        url = reverse("santa_public:postflight", args=(self.enrollment_secret.secret, uuid.uuid4()))
         response = self.post_as_json(url, {})
         self.assertEqual(response.status_code, 403)
 
     def test_postflight(self):
-        url = reverse("santa:postflight", args=(self.enrollment_secret.secret, self.enrolled_machine.hardware_uuid))
+        url = reverse("santa_public:postflight", args=(self.enrollment_secret.secret, self.enrolled_machine.hardware_uuid))
         response = self.post_as_json(url, {})
         self.assertEqual(response.status_code, 200)
         json_response = response.json()
         self.assertEqual(json_response, {})
+
+    def test_legacy_public_urls_are_disabled_on_tests(self):
+        data, serial_number, hardware_uuid = self._get_preflight_data()
+        routes = ['preflight', 'ruledownload', 'eventupload', 'postflight']
+
+        for route in routes:
+            with self.assertRaises(NoReverseMatch):
+                reverse(f"santa_public_legacy:{route}", args=(self.enrollment_secret.secret, hardware_uuid))
+            self.assertIsNotNone(reverse(f"santa_public:{route}", args=(self.enrollment_secret.secret, hardware_uuid)))
+
+    def test_mount_legacy_public_endpoints_flag_is_working(self):
+        data, serial_number, hardware_uuid = self._get_preflight_data()
```

Comment:
nitpick: data and serial number are not used in this test. You could simply use uuid.uuid4().

Reply:
Sure. I'll use the same `hardware_uuid` variable so the `uuid4` is always the same.